report_export.py

```diff
@@ -48,9 +55,10 @@
 for project in fw.projects.iter():
     pid = project.id
     name = project.label
-    print(f"Fetching view for project {name} ({pid})...")
-    d = json.load(fw.read_view_data(view, pid, decode=False, format="json-flat"))
-    all_data.extend(d)
+    if name =='CBTN_reports':
```

Comment:
it's much more computationally efficient if the for loop is removed & hard-code the `pid` and `name` for the `CBTN_reports` project (so then wont iterate through the numerous projects on Flywheel)

Reply:
Makes sense. 
Thanks for reviewing. Fixed the code.